.github/workflows/deploy_app_store.yml

```diff
@@ -97,3 +97,8 @@ jobs:
           ${{ env.DSYM_OUTPUT_PATH }}
       env:
         TAG_TYPE: App_Store
+
+    - name: Remove keychain
+      if: ${{ always() }}
+      run: bundle exec fastlane removeKeychain
+      continue-on-error: true
```

Comment:
Do we need `continue-on-error: true` here 🤔 That's because: 1, there's no subsequent step; 2. If the `removeKeychain` fails, the workflow still reports success for this step.

Reply:
> If the removeKeychain fails, the workflow still reports success for this step

I think this is what we are aiming for because we can ignore the error in this step, if this step can fail then the whole CI will show failure which is also confusing. What do you think?